static/js/people.js

```diff
@@ -256,6 +256,19 @@ export function reply_to_to_user_ids_string(emails_string) {
     return user_ids.join(",");
 }
 
+export function recipient_names_from_emails(emails) {
```

Comment:
This function name should either have `_string` appended to the end, to indicate it is doing a string join, or return an array (and have its caller do that). 

At present, it really sounds like it's going to give you an array, and someone trying to use it expecting that will likely be a bit sad :)

(I see the function just above this use the `_string` convention, which is why I suggested that).

Reply:
Sounds great! I will hopefully remember from hereon that we add `_string` to functions to make it clear that they return strings. Thanks :)

Reply:
Looks like you didn't actually get to this; I've reworked this commit to introduce `emails_to_full_names_string`.

Also used the commit message title "people: Extract emails_to_full_names_string."`

We like to use the "Extract" term for refactors that specifically extract a function. And we also avoid "refactor:" in the commit message prefix -- the point is to communicate what subsystem was changed (since the main purpose of that prefix is when trying to skip 200 commits to figure out what might have caused a regression -- and "refactor" isn't that relevant in that context, since while the intent may be to have no functional changes, most bugs are by accident!)

Reply:
Oh, you made these changes in the second commit. It kinda defeats the point of having two commits, if the second fixes an issue that had been in the first one :)

Reply:
Merged this prep commit as bc2149d1ef6b29e7be0aa3b492dbb249188843b1.

Reply:
Oh oops 😅 🙈  thanks!

That makes sense about the prefix! I will keep that in mind going forwards.